test/MUXControlsTestApp/master/AppBarButton-6.xml

```diff
@@ -0,0 +1,149 @@
+[Windows.UI.Xaml.Controls.Grid]
+    Padding=0,0,0,0
```

Comment:
Is there a script to copy the masters out of the pipeline? Or what is the procedure?

Reply:
Current procedure is you'd have to open that HelixTestFailureDumps.html file and click on the links to download. I guess I'll write another script to download and replace the old masters to make it easier/faster.

Reply:
Added a script in tools folder : )